### PR TITLE
PMM-7 fix upgrade annotation test

### PR DESCRIPTION
**Pull request by yurkovychv**

### CODE REVIEW

**Review by puneet0191 — APPROVED** [review]

File: tests/upgradePMM_test.js [hunk 1]
```diff
@@ -844,15 +844,14 @@ if (versionMinor >= 13) {
       const {
         serviceType, name, annotationName, dashboard,
       } = current;
-      const timeRange = 'Last 30 minutes';
+      const { service_name } = await inventoryAPI.apiGetNodeInfoByServiceName(serviceType, name, 'ssl');
+      const dashboardUrl = I.buildUrlWithParams(dashboard.split('?')[0], {
+        service_name,
+        from: 'now-30m',
```
Comment (puneet0191) [inline tests/upgradePMM_test.js:850]: change this to 5m

**Review by puneet0191 — CHANGES_REQUESTED** [review]

File: tests/upgradePMM_test.js (hunk 1 above)
Comment (yurkovychv) [reply tests/upgradePMM_test.js:850]: Are you sure? It may not show up if the upgrade will take more than 5 min

Comment (puneet0191) [reply tests/upgradePMM_test.js:850]: I realized we might be getting metrics from before the upgrade (30 mins is big time) if we use that large time offset, instead, we should be fetching the most recent monitored metrics

Comment (yurkovychv) [reply tests/upgradePMM_test.js:850]: We can chane it to 15 min and see if its stable enough

Comment (puneet0191) [reply tests/upgradePMM_test.js:850]: sure go for it

**Review by puneet0191 — APPROVED** [review]

**Pull request merged**